viv-wallet-app/src/App.vue

```diff
@@ -85,6 +86,7 @@ body {
     flex-direction: column;
     color: $white;
     text-align: left;
+    width: 155px;
```

Comment:
why not using your `m-X` variables?

Reply:
@maurelio1234 because m-X is for margins and paddings (small spacings).
  FYI I set a fixed width to avoid strangely effects (width change) when you select a different link.